server/api/attendance/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var express = require('express');
+var controller = require('./attendance.controller');
+var config = require('../../config/environment');
+var auth = require('../../auth/auth.service');
+
+var router = express.Router();
+
+// Basic Access / Manipulation
+router.get('/', auth.hasRole('mentor'), controller.index);
+router.get('/:id', auth.isAuthenticated(), controller.show);
+router.delete('/:id', auth.hasRole('mentor'), controller.destroy);
+
+// DON'T USE : does not generate required data for submission
```

Comment:
See above. It shouldn't be used normally, but is still useful for overriding....